### PR TITLE
frontend: move lightning config to context

### PR DESCRIPTION
**Pull request by thisconnect**
Moved Lightning config loader and subscription to a new lightning context/provider and hook, with this every component can now use the useLightning hook to get the current config.

This context could also be used to subscribe to NodeState changes.

### CODE REVIEW

File: frontends/web/src/app.tsx [hunk 1]
```diff
@@ -154,49 +153,49 @@ export const App = () => {
...
+                      devicesKey={devicesKey}
+                    />
+                    <RouterWatcher />
+                  </div>
+                  <Alert />
+                  <Confirm />
                 </div>
-                <Alert />
-                <Confirm />
-              </div>
```
Comment (thisconnect) [inline frontends/web/src/app.tsx:199]: With every provider there is always lots of lines that changed only due to indentation... for this reason I would prefer to merge https://github.com/digitalbitbox/bitbox-wallet-app/pull/2559 first, then master->staging-ln, and this PR

**Comment by thisconnect** [comment]
Tested that Lightning item in sidebar shows up, that the settings correctly show "activate lightning" / "deactivate lightning" when changing to fresh appfolder.dev.

**Review by Beerosagos — DISMISSED** [review]
untested LGTM

**Comment by thisconnect** [comment]
rebased

**Review by Beerosagos — APPROVED** [review]

**Pull request merged**